csi/util.go

```diff
@@ -25,16 +27,16 @@ const (
 	defaultStaleReplicaTimeout = 2880
 )
 
-// NewForcedParamsOsExec creates a osExecutor that allows for adding additional params to later occurring Run calls
-func NewForcedParamsOsExec(cmdParamMapping map[string]string) mount.Exec {
+// NewForcedParamsExec creates a osExecutor that allows for adding additional params to later occurring Run calls
+func NewForcedParamsExec(cmdParamMapping map[string]string) utilexec.Interface {
 	return &forcedParamsOsExec{
-		osExec:          mount.NewOSExec(),
+		exec:            utilexec.New(),
```

Comment:
Can you also make the exec/mount changes on the share-manager repository, since it give you a chance to go through the build process there.

Reply:
Sure. Added to my todo list